activation/poet_client_test.go

```diff
@@ -207,7 +207,7 @@ func TestPoetClient_CachesProof(t *testing.T) {
 
 	client, err := NewHTTPPoetClient(server, DefaultPoetConfig(), withCustomHttpClient(ts.Client()))
 	require.NoError(t, err)
-	poet := NewPoetServiceWithClient(db, client, DefaultPoetConfig(), zaptest.NewLogger(t))
+	poet := NewPoetServiceWithClient(db, client, DefaultPoetConfig(), zaptest.NewLogger(t), 1)
```

Comment:
I agree with @ivan4th, it would improve readability if we had a constant in the tests like `testTickSize` that is set to one and used everywhere instead of this 1 🙂